### DISCUSSION TITLE
Buildbase On School Server

### DISCUSSION THREAD

**Discussion by enemali** [Budibase/budibase#3450] [General]
iam a teacher and i just heard of Buildbase  and love it simplicity but i have these few questions 

1.  can Buildbase  run locally on the school server ?
2. if i use the online editor , can i export it to run locally or on my custom domain?
3. can Buildbase  connect to google-sheet or mysql 
4. does Buildbase  support video player component , 3d object viewer and drag and drop component? , these will be great for education app

**Comment by trashbrah** [comment]
> iam a teacher and i just heard of Buildbase  and love it simplicity but i have these few questions 
> 
> 
> 
> 1.  can Buildbase  run locally on the school server ?
> 
> 2. if i use the online editor , can i export it to run locally or on my custom domain?
> 
> 3. can Buildbase  connect to google-sheet or mysql 
> 
> 4. does Buildbase  support video player component , 3d object viewer and drag and drop component? , these will be great for education app

all 3 yes except 4 don't know four

**Reply by mjashanks** [reply]
Indeed - 
1. yes 
2. yes
3. MySQL not Sheets
4. It doesn't have any of these as a component, but does have an `Embed` component, where you can write any HTML/JS.
